packages/material-ui/src/styles/props.d.ts

```diff
@@ -134,10 +134,10 @@ export interface ComponentsPropsList {
   MuiDialogTitle: DialogTitleProps;
   MuiDivider: DividerProps;
   MuiDrawer: DrawerProps;
-  MuiExpansionPanel: ExpansionPanelProps;
-  MuiExpansionPanelActions: ExpansionPanelActionsProps;
-  MuiExpansionPanelDetails: ExpansionPanelDetailsProps;
-  MuiExpansionPanelSummary: ExpansionPanelSummaryProps;
```

Comment:
I think that we need to keep these types, otherwise, it's breaking.

Reply:
I reverted the typings changes. Just out of curiosity, now that the component itself is exported by the `MuiAccordion` these will not have any effect on the new component right?

Reply:
So just reverting the old components code with the console.error warnings should be fine, right?

Reply:
yes